frontend/src/styles/components/DiscussionPage.css

```diff
@@ -58,6 +58,12 @@
   border-bottom: 1px solid var(--color-light);
 }
 
+.snippet-wrapper:hover {
+  transition: 0.2s;
+  border-radius: 5px;
+  background-color: #313131;
```

Comment:
Why using random color? 
Don't we have variables for that? 
This will cause problem in Dark mode. 
Avoid using direct variables.

Reply:
i took the color from the notification bar. not random

Reply:
Either way it should from css variables.